docs/.vuepress/tools/jsdoc-convert/configuration.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
```

Comment:
Is there is a reason to keep the file in CommonJS format? All files within the module are refactored to `mjs` this is the last one.

Reply:
That way it is closer to json format, which I would like to use here. However, it doesn't work for me.

Reply:
You can export the config via defult export, like:
```js
const config = {
  pathToSource: '../../../../src',
  pathToDist: '../../../next/api',
  urlPrefix: '/next/api/',
  whitelist: [
    'dataMap/metaManager/metaSchema.js',
    'pluginHooks.js',
    'core.js',
    'translations/indexMapper.js',
    'editors/baseEditor/baseEditor.js',
    '3rdparty/walkontable/src/cell/coords.js',
    'plugins/copyPaste/focusableElement.js',
    'dataMap.js',
    'translations/maps/hidingMap.js',
    'translations/maps/indexesSequence.js',
    'translations/maps/trimmingMap.js',
    'utils/samplesGenerator.js',
    'translations/maps/physicalIndexToValueMap.js',
    'utils/ghostTable.js'
  ],
  seo: {
    'dataMap/metaManager/metaSchema.js': {
      title: 'Options',
      metaTitle: 'Options - API Reference - Handsontable Documentation',
      permalink: '/next/api/options'
    },
    'pluginHooks.js': {
      title: 'Hooks',
      metaTitle: 'Hooks - API Reference - Handsontable Documentation',
      permalink: '/next/api/hooks'
    },
    'core.js': {
      title: 'Core',
      metaTitle: 'Core - API Reference - Handsontable Documentation',
      permalink: '/next/api/core'
    },
    '3rdparty/walkontable/src/cell/coords.js': {
      title: 'CellCoords',
      metaTitle: 'CellCoords - API Reference - Handsontable Documentation',
      permalink: '/next/api/coords'
    }
  },
  linkAliases: {
    options: 'metaSchema',
    hooks: 'pluginHooks'
  }
};

export default config;
```
Then I run `npm run docs:api`. I haven't noticed any issues.

Reply:
done 436f5ca4ca416de4c942f851a9d2476c08e3089a